src/lib/mina.ts

```diff
@@ -307,6 +316,29 @@ function LocalBlockchain({
     sendTransaction(txn: Transaction) {
       txn.sign();
 
+      let commitments = Ledger.transactionCommitments(
+        JSON.stringify(zkappCommandToJson(txn.transaction))
+      );
+
+      txn.transaction.accountUpdates.forEach(async (party) => {
+        try {
+          let account = ledger.getAccount(
+            party.body.publicKey,
+            party.body.tokenId
+          );
+          if (account) {
+            await verifyAccountUpdate(
+              account!,
+              party,
+              commitments,
+              proofsEnabled
+            );
+          }
```

Comment:
Should we somehow merge these errors with the ones in this next `try/catch/finally` block rather than skipping them (if `account` is falsey) or console.log-ing (if the error is caught)

Reply:
Is there a reason to not just throw the error here? Do we want to make sure other problems with the transaction are logged as well? I'm OK if we don't

Reply:
after thinking about it, I would agree its probably better to just throw right away